bin/configs/jaxrs-spec-ee.yaml

```diff
@@ -0,0 +1,11 @@
+generatorName: jaxrs-spec
+outputDir: samples/server/petstore/jaxrs-spec
```

Comment:
i think the output folder cannot be the same. please change it to `samples/server/petstore/jaxrs-spec-ee` instead

Reply:
please also update [.github/workflows/samples-jaxrs.yaml](https://github.com/OpenAPITools/openapi-generator/blob/037b41e1739ad7071afb875388b09bbd317ae67c/.github/workflows/samples-jaxrs.yaml) with the new output directory